tests/unit/test_character_serialization.py

```diff
@@ -1,10 +1,8 @@
-
-
 from nucypher.characters.lawful import Ursula
 
 
-def test_serialize_ursula(federated_ursulas):
-    ursula = federated_ursulas.pop()
+def test_serialize_ursula(ursulas):
```

Comment:
If we modify this test to just create a simple Ursula OR move this test to integration, then the timing for `tests/unit` tests goes down from ~23s to 3s.

Reply:
🚀 👌